Resources/JavaScript/singleRound.js

```diff
@@ -0,0 +1,32 @@
+//Import other functions
+import playerChoice from "./playerChoice.js";
+import getComputerChoice from "./getComputerChoice.js";
+
+//Play singleRound
+export default function singleRound() {
+  let playerSelection = playerChoice();
+  let computerSelection = getComputerChoice();
+
+  if (playerSelection === computerSelection) {
+    console.log(`It's a tie!`);
+    return "Tie";
+  } else if (
+    (playerSelection === "rock" && computerSelection === "scissors") ||
+    (playerSelection === "paper" && computerSelection === "rock") ||
+    (playerSelection === "scissors" && computerSelection === "paper")
+  ) {
+    console.log(`You Win! ${playerSelection} beats ${computerSelection}`);
```

Comment:
Pay attention to detail, this is not the response format, read through the instructions again

Reply:
Sorry boss, I misinterpreted your instruction, I thought when you said 'return a string that declares the winner of the round like so: "You Lose! Paper beats Rock"', that we can use something like that.